apps/table-test/src/app/wrapper/wrapper.component.ts

```diff
@@ -1,13 +1,15 @@
-import { CommonModule } from '@angular/common';
 import { Component, Input } from '@angular/core';
 
 @Component({
 	selector: 'app-wrapper',
-	imports: [CommonModule],
+	imports: [],
```

Comment:
It's in the test so it doesn't **have** to be fixed, but I'd remove this empty imports array.